cmd/podman/validate/args.go

```diff
@@ -65,7 +65,7 @@ func CheckAllLatestAndPodIDFile(c *cobra.Command, args []string, ignoreArgLen bo
 }
 
 // CheckAllLatestAndIDFile checks that --all and --latest are used correctly for containers and pods
-// If idFileFlag is set is set, also checks for the --cidfile or --pod-id-file flag.
+// If idFileFlag is set, also checks for the --cidfile or --pod-id-file flag.
```

Comment:
In this line what does `also checks for the --cidfile or --pod-id-file flag` means ? In following function i can't see if following flags are used, might as well correct this entire line if these two flags are not valid here.

Reply:
IMHO the comment is correct. If the argument `idFileFlag` takes one of the values `cidfile` or `pod-id-file`, the function checks, among other things, that `latest` or `all` have not also been specified.